<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSearchWindow</class>
 <widget class="QMainWindow" name="DataSearchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Search</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Date field (/- all)</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="1">
       <widget class="QComboBox" name="date_cb">
        <property name="font">
         <font>
          <pointsize>27</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="dateNotChk">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Category (* - all)</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="catNotChk">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="category_cb">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>*</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Note field (* - all)</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="noteNotChk">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="note_cb">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>*</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="sumLabel">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sum= 0.0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="sWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>586</width>
         <height>372</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnFilter">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
